CONTRIBUTING.md

```diff
@@ -4,18 +4,33 @@ Thanks for your interest in contributing to rust-landlock!
 
 ## Testing vs kernel ABI
 
-The Landlock functionality exposed differs between kernel versions.  In order
-to test all possible variations, the rust-landlock tests will run different
-subsets of tests based on the landlock support in the current kernel.
+The Landlock functionality exposed differs between kernel versions. Based on
+the Landlock ABI version of the running system, rust-landlock runs different
+subsets of tests. For local development, running `cargo test` will test against
+your currently running kernel version (and the Landlock ABI that it ships).
+However, this may result in some tests being skipped.
 
-In order to fully test a change, it should be verified against a range of
-kernel versions.  This is done by the github actions CI, but is currently
-challenging to do locally.  For local development, running `cargo test` will
-test against your currently running kernel version, which may result in some
-tests being skipped.
+To fully test a change, it should be verified against a range of ABI versions.
+This is done by the Github Actions CI, but doing so locally is challenging.
 
-The kernel to test against can be overridden using the LANDLOCK_CRATE_TEST_ABI
-environmental variable.  For more details see the comment in
-`compat.rs:current_kernel_abi()`.
+Using the `LANDLOCK_CRATE_TEST_ABI` variable, the tested ABI version can be
+overridden. For more details, take a look at the comment in
+[`compat.rs:current_kernel_abi()`][currentkernelabi].
 
-For more information about Landlock ABIs see https://landlock.io/rust-landlock/landlock/enum.ABI.html
+For more information about Landlock ABIs, see: [landlockabi]
+
+[currentkernelabi]: https://github.com/landlock-lsm/rust-landlock/blob/main/src/compat.rs
+[landlockabi]: https://landlock.io/rust-landlock/landlock/enum.ABI.html
+
+## Licensing & DCO
+
+rust-landlock is double-licensed under the terms of [the Apache 2.0 license][apache-2.0]
+and the terms of [the MIT license](https://spdx.org/licenses/MIT.html).
+
+All changes submitted to rust-landlock must be signed off, in line with standards
+such as those of [the Linux kernel][linux-dco].
```

Comment:
"Such as" makes it sound like maybe sign-off means some other standards.  I feel like for something like signed-off, being as unambiguous as possible and just saying "in line with the standards of the Linux kernel" makes more sense to me.